src/index.html

```diff
@@ -5,8 +5,349 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <picture class="header__logo">
+        <a href="#home"><img src="./images/logo.png" alt="mate-logo"></a>
+      </picture>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="is-active nav__link" href="#">apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">smartphones</a></li>
+          <li class="nav__item">
```

Comment:
Add blank line between the links for more readable
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="#">smartphones</a>
          </li>
          
          <li class="nav__item">
```